app/views/items/show.html.erb

```diff
@@ -5,7 +5,7 @@
 </head>
 <body>
 	<div class = "container">
-	<h4><%= link_to "Continue shopping", items_path %></h4>
+	<h4><%#= link_to "Continue shopping", items_path %></h4>
```

Comment:
@GabrielAfflitto wanna update this one to the root_path?

Reply:
Updated!